Felienne/hedy
# [UI IMPROVEMENT] Adds warning when leaving class customization without saving

**Description**
In this PR we add a confirmation pop-up to the class customizations page when a user tries to leave the page without saving their changes. Nice improvement to the current "program unsaved" approach is that the warning is always returned, independent of the way the user decides to leave the page. So also when directly typing a new URL, using the back button of the browser and so on.

**Fixes**
This PR fixes #2866.

**How to test**
Make sure you are a teacher, customize a class but don't save (yet). Validate that you receive a confirmation pop-up when clicking the "back to class" button to verify that you are sure you want to leave.
